README.md

```diff
@@ -52,6 +52,31 @@ service and time to live.
 
 <!-- MDOC !-->
 
+## Configuration
+
+Jackalope puts the publish commands on a work list before it sends them to Tortoise311.
+The work list moves the commands from waiting to be sent, to pending (sent and waiting for a response),
+to discarded when confirmed by Tortoise311 as processed or when they are expired.
+
+The work list has a maximum size which defaults to 100. Only a maximum number of publish commands
+can wait, should Tortoise311 be temporarily disconnected, to be forwarded to Tortoise311.
+
+The waiting and pending publish commands can survive a reboot but only if the persistent work list
+implementation is used. The default is the transient work list implementation.
+
+To use the persistent implementation, you can add this to `config.exs`
+
+```elixir
+config :jackalope, 
+  :work_list_mod, Jackalope.PersistentWorkList
+
+```
```

Comment:
Is there any reason to use the Application environment for this? Every other configuration option goes through `Jackalope.start_link/1`?

```suggestion
```